proto/reaction.proto

```diff
@@ -730,6 +859,9 @@ message ReactionAnalysis {
   // Whether an internal standard was used with this analytical technique for
   // quantification, e.g., of yield.
   Boolean.BooleanValue uses_internal_standard = 8;
+  // Whether an authentic sample of a material was used for callibrating
```

Comment:
What does "authentic" mean in this context?

Reply:
Authentic refers to a sample of a compound whose structure has been proven by the accepted means (1H NMR, 13C NMR, IR, HRMS), and is now being used as a comparison point for other analytical methods. This way one can determine if that compound is present in a reaction in the future by quicker methods (e.g., TLC or LCMS) that are not acceptable "proof" of a structure by themselves.

Reply:
Let's expand the comment to be a little more explicit. Even if synthetic chemists understand, it would be useful for the informaticians to have this explanation

Reply:
Done.